plugins/replicate.js

```diff
@@ -28,7 +24,7 @@ var streamErrors = {
   'read ETIMEDOUT': true,
   'write ECONNRESET': true,
   'write EPIPE': true,
-  'stream is closed': true, // rpc method called after stream ended
+  'stream is closed': true // rpc method called after stream ended
```

Comment:
fwiw, the trailing comma was deliberate, so that future commits adding or removing lines could have cleaner diffs (as they would not have to add a comma to the last line)